### DISCUSSION TITLE
有谁用过 OpenZeppelin Defender

### DISCUSSION THREAD

**Discussion by liushooter** [rebase-network/hello-world#30] [General]
有谁用过 [OpenZeppelin Defender](https://openzeppelin.com/defender/)， 欢迎来 Rebase大学分享使用体验啊

**Comment by Skyge** [comment]
我只简单监控了一个合约的方法，看起来还行，好像还有更复杂的操作，好像是可以满足条件触发操作，没试过。

**Reply by liushooter** [reply]
我挪了1个使用教程放到了 B站：https://www.bilibili.com/video/BV1yP4y1w7kQ/ ， 还没有实操。

来rebase 分享下啊  😄

**Comment by liushooter** [comment]
https://www.bilibili.com/video/BV1yP4y1w7kQ?t=794

OpenZeppelin Defender 可以运行自动任务，比如 **每10 分钟自动 mint 10个nft**，https://docs.openzeppelin.com/defender/autotasks-api-reference
